### DISCUSSION TITLE
Android Ipad Controller.  Mouse works Keyboard doesn't

### DISCUSSION THREAD

**Discussion by gustavorp75** [AppGround-io/bluetooth-keyboard-and-mouse-support#231] [Q&A]
Hi,  Been looking for an App like this for a while.  Seems like this is what I need, but i'm having issues.  I can connect from my Ultra 22 to my Ipad 8th Gen via bluetooth and the mouse portion works great.  Keyboard inputs do not work.  Is there a known issue here?  Would like to get this to work.

**Comment by Kaptensvart** [comment]
I'm assuming u meant writing from the app to your iPad. I just tried it and it worked for me
